playground/src/kata.tsx

```diff
@@ -40,7 +44,7 @@ function ExplainedSolutionElem(props: { solution: ExplainedSolution }) {
               </pre>
             );
           case "text-content":
-            return <Markdown markdown={item.asMarkdown}></Markdown>;
+            return <Markdown markdown={item.content}></Markdown>;
```

Comment:
I can't recall, why do we use markdown in the playground instead of html? If the playground could use the same setup as the viewer we could stop producing the markdown version.

Reply:
I cover that in the first paragraph of the PR. We may want to look at doing that later.